### PR TITLE
Movie seats booking

### PR DESCRIPTION
**Pull request by AnnaStoyano**
Please, review) 
[Demo](https://annastoyano.github.io/Movie_seat_booking/.) 
[Code](https://github.com/AnnaStoyano/Movie_seat_booking)

### CODE REVIEW

File: submissions/AnnaStoyano/Movie_seats_booking/index.html [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+                </ul>
+            </nav>
+        </div>
+    </header>
+    <main>
+        <article class="film-poster">
+            <div class="film-info">
+                <h2 class="film-title">
+                    Palace for Putin. The history of the biggest bribe
+                </h2>
```
Comment (A-Ostrovnyy) [inline submissions/AnnaStoyano/Movie_seats_booking/index.html]: Mabey `h1` is the preferable choice for this element.

File: submissions/AnnaStoyano/Movie_seats_booking/index.html [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+                    no one is allowed. We will visit Putin. Let's make sure with our own eyes that this person is in
+                    his craving for luxury, for wealth, he completely lost his mind. We will find out with whose money
+                    and how this
+                    luxury. And how, over the past 15 years, the biggest bribe in history has been given and the most
+                    dear palace in the world.
+
+                </p>
+                <div class="rating__wrapper">
+                    <img class="star" src="./images/star.svg" alt="rating star icon">
+                    <strong class="raiting">IMDb: 9.8/10</strong>
```
Comment (A-Ostrovnyy) [inline submissions/AnnaStoyano/Movie_seats_booking/index.html:44]: All visual representing must be in CSS not in HTML. I mean if someone changed this tag inner text still be bold?

File: submissions/AnnaStoyano/Movie_seats_booking/index.html [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+                </div>
+            </div>
+            <img class="film_img" src="./images/film_logo.jpg" alt="Putin's Palace">
+        </article>
+
+        <section class="timeslot">
+            <h3 class="timeslot__title title">Time</h3>
+            <ul class="timeslot__list flex">
+                <li class="timeslot__item">
+                    <button name="time" class="time_btn" value="11:00">11:00</button>
```
Comment (A-Ostrovnyy) [inline submissions/AnnaStoyano/Movie_seats_booking/index.html]: For date and time preferable to use `<time>` tag [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
Extra like for movie choice.
See my comments below.

File: submissions/AnnaStoyano/Movie_seats_booking/index.html (hunk 2 above)
Comment (AnnaStoyano) [reply submissions/AnnaStoyano/Movie_seats_booking/index.html:44]: No, I had missed this case but I fixed it. Thanks for advise)

**Comment by AnnaStoyano** [comment]
@A-Ostrovnyy Thanks for your notes) Can you see again, please?)

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**